src/benchmarks/micro/runtime/StoreBlock/StoreBlock.tt

```diff
@@ -0,0 +1,114 @@
+﻿<#@ template debug="false" hostspecific="true" language="C#" #>
+<#@ output extension=".cs" #>
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+
```

Comment:
nit: the old benchmarks from the xunit-performance era have no namespace for historical reasons and usually we add namespaces for new benchmarks